docs/intro/go.md

```diff
@@ -0,0 +1,95 @@
+---
+order: 2
+description: Install the Go programming language
+---
+
+# Install Go
+
+In this tutorial, you will install the Go programming language (Golang) on your local computer. Follow the instructions for your operating system:
+
+## macOS
+
+* Download the latest MacOS installer package from <https://golang.org/dl/>.
+* Open the downloaded package and follow the prompts through to completion.
+* By default, the package installs the Go distribution to `/usr/local/go`, however it is always best to define the path explicitly:
+
+    1. Open or create a `~/.profile` file with your favorite command-line text editor.
+    2. Add the following lines:
+
+        ```sh
+        export PATH=$PATH:$(go env GOPATH)/bin
+        ```
+
+    3. To make sure these changes are applied, run the following command:
+
+        ```sh
+        source ~/.profile
+        ```
+
+* Verify the installation of go by checking its version:
+
+    ```sh
+    go version
+    ```
+
+## Linux
+
+* Download the latest Linux distribution package from <https://golang.org/dl/>.
+* Extract the archive you downloaded into `/usr/local` using the following command:
+
+```sh
+sudo tar -C /usr/local -xzf go1.16.6.linux-amd64.tar.gz
+```
+
+**Note - Make sure to install the latest version available at that time.**
```

Comment:
```suggestion
**Note:** Make sure to install the latest supported version.
```

Reply:
I'm so glad to see this Note, I've updated our technical style guidance on Notion to include this standard. At Tendermint, we bold only the word "Note" and then use plain text for the rest of the sentence, per https://developers.google.com/style/notices